documentation/docs.html

```diff
@@ -0,0 +1,127 @@
+                    style="width: 880px; height: 322px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                    <br><br>
+                    <p class="font-1_5x">-Retina Avatar</p>
+                    <br><br>
+                    <div class="retina-avatar">
+                        <img class="retina sm" src="/assets/images/retina-image.jpg" alt="avatar-image">
+                        <img class="retina md" src="/assets/images/retina-image.jpg" alt="avatar-image">
+                        <img class="retina lg" src="/assets/images/retina-image.jpg" alt="avatar-image">
+                        <img class="retina xl" src="/assets/images/retina-image.jpg" alt="avatar-image">
+                        <img class="retina xxl" src="/assets/images/retina-image.jpg" alt="avatar-image">
+                    </div>
+                    <br><br>
+                    <iframe class="code-snippet"
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=49px&dsblur=62px&wc=true&wa=true&pv=25px&ph=38px&ln=false&fl=1&fm=Hack&fs=14px&lh=170%25&si=false&es=2x&wm=false&code=%253Csection%2520class%253D%2522avatar%2520flex%2522%253E%250A%2520%253Cdiv%2520class%253D%2522retina-avatar%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522retina%2520sm%2522%2520src%253D%2522%252Fassets%252Fimages%252Fretina-image.jpg%2522%2520alt%253D%2522avatar-image%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522retina%2520md%2522%2520src%253D%2522%252Fassets%252Fimages%252Fretina-image.jpg%2522%2520alt%253D%2522avatar-image%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522retina%2520lg%2522%2520src%253D%2522%252Fassets%252Fimages%252Fretina-image.jpg%2522%2520alt%253D%2522avatar-image%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522retina%2520xl%2522%2520src%253D%2522%252Fassets%252Fimages%252Fretina-image.jpg%2522%2520alt%253D%2522avatar-image%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522retina%2520xxl%2522%2520src%253D%2522%252Fassets%252Fimages%252Fretina-image.jpg%2522%2520alt%253D%2522avatar-image%2522%253E%250A%2520%253C%252Fdiv%253E%250A%253C%252Fsection%253E"
+                    style="width: 880px; height: 322px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
```

Comment:
You can maintain indentation, either manually or prettier extension in VScode

Reply:
Done